### PR TITLE
Update header and tabs for Storybook

### PR DESCRIPTION
**Pull request by adriana-alter**
Resolves https://jira.nypl.org/browse/SIMPLY-2175 (update the unstyled Storybook pages before sending our Storybook docs to the design system forum).

Before:
Screen Shot 2019-07-30 at 1 08 11 PM
Screen Shot 2019-07-30 at 1 08 44 PM

<hr />

After:
Screen Shot 2019-07-30 at 1 08 20 PM
Screen Shot 2019-07-30 at 1 09 03 PM

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -6,25 +6,32 @@ export interface HeaderProps {
...
 export default class Header extends React.Component<HeaderProps, {}> {
+  static defaultProps = {
+    loggedIn: false,
+  };
+
   render(): JSX.Element {
     let src = this.props.imgSrc ? require(`${this.props.imgSrc}`) : "";
     return(
-      <nav className="navbar navbar-default">
+      <nav className="reusable-navbar navbar-default clearfix">
```
Comment (EdwinGuzman) [inline src/components/Header.tsx]: I did not update this in the other app because I was going to do it here, but since you're working on it... this should be a `<header>` element. The `nav` element is only for navigation-related elements. This is tricky but I think there should be no `nav` in this component. Right now it only has a title and a logo so those are not part of it. You can argue that the logout link should be surrounded by a `nav` but it's not always there so I think that's enough to exclude the `nav` element here.

File: src/components/Tabs.tsx [hunk 2]
```diff
@@ -61,12 +58,12 @@ export default class Tabs extends React.Component<TabsProps, TabsState> {
     items.map((item, idx) => {
       let [name, data] = item;
       let current = idx === this.state.tab;
-      let navItem: JSX.Element = (
+      let navItem = (
         <li key={name} role="presentation" className={`tab-nav ${current ? "current" : ""}`}>
           <button
```
Comment (EdwinGuzman) [inline src/components/Tabs.tsx]: Do you want to use the `Button` component here?

File: src/components/Tabs.tsx [hunk 3]
```diff
@@ -81,7 +78,7 @@ export default class Tabs extends React.Component<TabsProps, TabsState> {
 
       navs.push(navItem);
 
-      let contentItem: JSX.Element = (
```
Comment (EdwinGuzman) [inline src/components/Tabs.tsx]: Why did you remove these types?

File: src/components/Tabs.tsx [hunk 4]
```diff
@@ -102,7 +99,7 @@ export default class Tabs extends React.Component<TabsProps, TabsState> {
   render(): JSX.Element {
     let [navs, content] = this.makeTabs();
     return (
-      <section className="tabs">
+      <section className="reusable-tabs">
```
Comment (EdwinGuzman) [inline src/components/Tabs.tsx]: I think `tabs` is fine. If you use `reusable-` here then it'd be good to use the same namespace for the other components to have a consistent naming convention.

Comment (adriana-alter) [reply src/components/Tabs.tsx]: The point was to be able to go ahead and use the styling on Storybook even though we haven't yet implemented this component in the interfaces; the interfaces already have things with className `tabs`, which we don't want this to overwrite.  (We had commented out the import statement for `tabs.scss` in `app.scss` in order to prevent this...but that's why the styling wasn't being used in Storybook.)  In other words, changing the class names is just a quick fix so that we can send the Storybook link to the design system forum ASAP; afterwards, we can reassess what kind of naming convention we want.

File: src/components/Tabs.tsx (hunk 2 above)
Comment (adriana-alter) [reply src/components/Tabs.tsx]: In general: yes, definitely.  But I think that's a different ticket.

File: src/components/Tabs.tsx (hunk 4 above)
Comment (EdwinGuzman) [reply src/components/Tabs.tsx]: Right, and I think those `tab` classes come from bootstrap. Next time let me know that's the goal for the PR or mention that it's temporary.

Comment (adriana-alter) [reply src/components/Tabs.tsx]: Full explanation is in the linked JIRA ticket.

File: __tests__/components/Header-test.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ describe("Header", () => {
     expect(text.text()).to.equal("Test Header!");
   });
   it("should render a logout button", () => {
-    let logout = wrapper.find(".navbar-btn");
+    let logout = wrapper.find(".header-btn");
```
Comment (EdwinGuzman) [inline __tests__/components/Header-test.tsx]: Eventually, this should be more specific in case you have more buttons in the header.
